eng/tools/TypeSpecValidation/package.json

```diff
@@ -13,6 +13,11 @@
     },
     "devDependencies": {
         "@types/debug": "^4.1.8",
-        "@types/node": "^18.16.18"
+        "@types/node": "^18.16.18",
+        "tslib": "^2.6.0",
```

Comment:
How is tslib used?

Reply:
If I remove this, compile fails with this error:

```
~/specs-mh/eng/tools/TypeSpecValidation$ npm i --no-package-lock

> @azure-tools/typespec-validation@0.0.1 postinstall
> tsc

src/TypeSpecValidation.ts:10:16 - error TS2354: This syntax requires an imported helper but module 'tslib' cannot be found.

10 async function runCmd(cmd: string, cwd: string) {
                  ~~~~~~
```

I think it's coming from here:

https://github.com/Azure/azure-rest-api-specs/blob/f6bf6555a71cb3167dcff04cc7964bda9ae36a88/tsconfig.json#L19

Reply:
ah, not sure that options needs to be true then

Reply:
Do you recommend changing this in the root `tsconfig.json`?  I don't know what impact this might have (since it impacts the whole repo), but I can open an issue to try it.

Reply:
Even better, I think I can override the option to false for just my tool.

Reply:
hhm, I guess I have never used this option ever, seems like it is really just meant to reduce a little bit duplication in the compiled javascript. I am not sure it is very valuable if you are targeting es6 modules and the latest node features where typescript doesn't have to do a ton of duplication.
Think for this tool as you are not publishing it anyway its probably even less worth to include.

Reply:
I am also guessing it should really be a dependency not a dev dependency if you need to include it.

Reply:
Fixed by setting `importHelpers:false` in the tool `tsconfig.json`.

Reply:
is there any other typescript tool yet in the spec repo?

Reply:
I think it might be used by this file:

https://github.com/Azure/azure-rest-api-specs/blob/f6bf6555a71cb3167dcff04cc7964bda9ae36a88/scripts/prettier-swagger-plugin.ts

As documented here:

https://github.com/Azure/azure-rest-api-specs/blob/f6bf6555a71cb3167dcff04cc7964bda9ae36a88/documentation/ci-fix.md#prettier-check

Reply:
I would maybe file an issue to investigate if setting `importHelpers: false` at the root would just not be better for everyone. I don't see tslib being referenced anywhere

Reply:
Tracking issue: #24766